byron/ledger/executable-spec/test/Ledger/Pvbump/Properties.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Ledger.Pvbump.Properties where
+
+import           Control.State.Transition (applySTS, TRC(..))
+import           Generator
+import           Hedgehog
+import           Ledger.Update (PVBUMP)
+
+
+-- Property #1 for the PVBUMP STS
+-- 
+-- Given an empty list of mappings from a slot to a pair of a protocol
+-- version and protocol parameters (this is the fads part of the
+-- environment), the PVBUMP STS results in the same state it started
+-- in.
+emptyPVUpdate :: Property
+emptyPVUpdate = property $ do
+  PVBUMPEnv env  <- forAll pvbumpEmptyListEnvGen
+  PVBUMPState st <- forAll pvbumpStateGen
+  sig            <- forAll pvbumpSignalGen
+  case applySTS @PVBUMP $ TRC (env, st, sig) of
```

Comment:
Why generating the signal instead of simply writing:

```haskell
  case applySTS @PVBUMP $ TRC (env, st, ()) of
```

Reply:
Obviously either would work. The way it is now shows that these values need not be hard-coded. If you're keen on seeing what you proposed, I'll change it.

Reply:
`PVBUMP` signals can only be `()`. Personally I prefer to avoid writing unnecessary code, but do whatever you think is sensible.

Reply:
`()` it is.